openshift/ci-operator/build-image/Dockerfile

```diff
@@ -22,6 +23,3 @@ RUN GOFLAGS='' go install github.com/mikefarah/yq/v3@latest
 # go install creates $GOPATH/.cache with root permissions, we delete it here
 # to avoid permission issues with the runtime users
 RUN rm -rf $GOPATH/.cache
-
-# Allow runtime users to add entries to /etc/passwd
```

Comment:
hm, no idea why that was here in the first place. Ideas @skonto @mgencur?

Reply:
it's a legacy setting used only in SO Dockerfiles and we don't need it anymore

Reply:
It shoul dbe moved to the CI config or something, see [here](https://redhat-internal.slack.com/archives/CD87JDUB0/p1726045452787339).